CONTRIBUTING.adoc

```diff
@@ -5,6 +5,42 @@ In the spirit of free software, _everyone_ is encouraged to help improve this pr
 To contribute code, simply fork the project on GitHub, hack away and send a pull request with your proposed changes.
 Feel free to use the https://github.com/asciidoctor/atom-language-asciidoc/issues[issue tracker] to provide feedback or suggestions in other ways.
 
+== Project organization
+
+The project is hosted and run using GitHub.
+The issue tracker is in place to report bugs, discuss improvements, show our ambitious goals and desires for the future, and encourage each other.
+Classification labels and versioning tags are added for clarity and the ability to search
+
+=== Labels
+
+Labels are added to issues and pull request for clarity, in the present but also when looking back at previous issues and pull requests.
+
+.Type labels
+|===
+|bug         | the current implementation has a flaw that needs to be resolved
+|question    | generic question
+|improvement | refinement on top of an existing feature
+|enhancement | new feature
+|duplicate   | a similar (open) issue already exsists
+|invalid     | not, or no longer, true
+|declined    | not accepted for inclusion
```

Comment:
I recommend changing the description of declined to "a decision was made not to pursue". That keeps the door open for someone who wants to look into it.